### ISSUE TITLE
[Enhancement] UI/UX renewal

### ISSUE THREAD

**Issue by pkzxs** [pkzxs/Aurora.Music#26]
Hi everyone, I'm planning to make a bit layout change in Aurora Music, since some pages are not perfect yet, and the sidebar is still not quite easy to use...

Thus I opened this issue to collect some of your ideas to make a more fluent appearance of Aurora Music, I'll post some idea below, please give +1 or -1 to show your prefers. Obviously, don't hesitate to put your own idea in this issue. All of us will pick the best!

**Comment by pkzxs** [comment]
What do you think of this layout:

![](https://i.loli.net/2018/04/24/5adef888f173d.png)

which the playing panel is placed at bottom and always visible, and the ham panel is opened

![](https://i.loli.net/2018/04/24/5adef98db2c97.png)

**Comment by pkzxs** [comment]
Here's a new layout created by a nice guy:

![](https://i.loli.net/2018/04/24/5adf4bc5c3a5e.png)

What have changed:
* Remove the acrylic on the main area for a white background
* Change the opacity of the title "Good Morning"
* Change the hamburger menu (HM) with a left bar for navigation 
    * Make it large
    * Add the label for each icon under its icon
* Center the search icon, and replace it by a custom search bar
* Change the position (z axis) of the player bar to appear at lvl 0 
    * layer 0 : navigation & player bar
    * layer 1 : main area (with the content)
* Change a bit the design of the player bar.

**Comment by dakehero** [comment]
I prefer the new UI

新UI更好看

**Comment by lionnexx** [comment]
UI with more Acrylic is better

**Comment by biqiu-ssw** [comment]
New UI are beautifui,old search box is discord

**Comment by lionnexx** [comment]
I want to ask, In new Hamburger menu so little space for the titles of category (and they always visible), in some localisations (Russian for example) very long typing titles. 
They may not fit. In English all looking good.

**Comment by pkzxs** [comment]
Hi @lionnexx1986  @biqiu-ssw  @dakehero  I've finished(may be not) renewal of app, please check if there's bugs of displaying text or interaction. Thanks a lot 😄

**Comment by lionnexx1986** [comment]
Hi @pkzxs! I already tested! It seems all looking and works good! Good update! thank you very much for the work! Very needed new nice renewal of mini player and black theme! Keep it up!

**Comment by biqiu-ssw** [comment]
If FM,Lib,Home also use fluent design like about will be better
主页，曲库，fm等如果像关于界面一样使用fluent design的模糊效果或许会更好点？
![about](http://ot6z44ono.bkt.clouddn.com/about.png)
![home](http://ot6z44ono.bkt.clouddn.com/home.png)
![lib](http://ot6z44ono.bkt.clouddn.com/lib.png)

**Comment by pkzxs** [comment]
@biqiu-ssw  还没有全部更新，我还在考虑如何能把子页面变得更契合一些

**Comment by lionnexx1986** [comment]
@pkzxs If use Fluent from desktop wallpaper like it used in FM will be more better/ And it will be great if Fluent will be more transparent
 In about is constant picture.
![2018-05-15](https://user-images.githubusercontent.com/39178626/40031764-c576a498-57f9-11e8-98ca-79c6b8776d8e.png)
![2018-05-15](https://user-images.githubusercontent.com/39178626/40031821-1ab47200-57fa-11e8-9751-d0f06f0c23f6.png)

**Comment by lionnexx1986** [comment]
Hi @pkzxs! Wow! Nice new mini player! Im very like it! But please make option in setting "Behind all windows" or something like that. As for me very important thing. Please!
About Dark theme. It is cool that you add Dark theme but it is dont work (
Thank you for you work! From me donat!)

**Comment by lionnexx1986** [comment]
Hi @pkzxs! Black theme work and looks very good! 
But there are a couple of nuances
1 This white frame on main window, looks very discord
![2018-05-19](https://user-images.githubusercontent.com/39178626/40270242-11f0e8ce-5b92-11e8-9eec-87b27fb8cf88.png)

2. In now playoing screen nothing see. Seems becouse the accent color, 
![2018-05-19 1](https://user-images.githubusercontent.com/39178626/40270251-38b96dd2-5b92-11e8-8f13-a85390ef536a.png)

Thank you very much!

**Comment by lionnexx1986** [comment]
@pkzxs In the albums, artist menu have bad visibility artist names. 
![2018-05-19 2](https://user-images.githubusercontent.com/39178626/40271395-6e7acaca-5ba5-11e8-97ec-6002bce5000c.png)

**Comment by pkzxs** [comment]
Hi @lionnexx1986 very thanks for your correction, I will soon fix these 👍

**Comment by lionnexx1986** [comment]
@pkzxs You done well and making a very good music player! 👍 
other than these corrections, Please make mini player function "behind all windows" in option, and make mini player saving size and position. Im very offen use mini player, and this two functions very necessary.
Wil be waiting next update!

**Comment by pkzxs** [comment]
@lionnexx1986 Oh yes, I'll try to add it. Due to the limitation of the window manage API, I think it may not work so well as expectation, But I'll work hard to find if it can be done!

**Comment by lionnexx1986** [comment]
@pkzxs oh i got it thank for explain. I have few more proposition for you.
1 Genres in album category
2 Increasing rewind bar click zone (you now have to small, need to target, exacly in the bar)
3 Increasing artist picture and album picture (too small now, like i think)
![2018-05-18 1](https://user-images.githubusercontent.com/39178626/40274355-59377504-5bdd-11e8-9480-92ff3d933a8b.png)
![2018-05-18](https://user-images.githubusercontent.com/39178626/40274356-5954616e-5bdd-11e8-9727-345eeb7b85f5.png)

And thats all i think...

**Comment by lionnexx1986** [comment]
@pkzxs Well, about everything in order
1. Appears strange Effect in All Covers (Like Decreased Brighteness)
![2018-05-20](https://user-images.githubusercontent.com/39178626/40277181-88eee94e-5c23-11e8-9f0e-326394d4ef17.png)
2 You work well with mini player! But then i exit from mini player, resets main window position, I think it is have to be, I can move it, but i cant! Need to maximize window first, then minimize, and then main windows moving
Thank you!

**Comment by OpportunityLiu** [comment]
@lionnexx1986 
Agreed, dark theme is not night mode, should not apply dark cover on images.  

Another problem is that the fly-out will not close automatically after navigation.
![image](https://user-images.githubusercontent.com/13471233/40287352-60c49e8e-5cdf-11e8-889e-5bd04711c64f.png)

**Comment by xperiandri** [comment]
Make some padding at the right.
So that opaque area does not look sticking to the right.

**Comment by lionnexx1986** [comment]
Hi @pkzxs, im just sending to you 3 cuo of cofee) Thankfully for your job! Im like your player!
Can you please make that main window and mini player save their positions! It will be very good!
![2018-05-22](https://user-images.githubusercontent.com/39178626/40331798-25e5e1f8-5d5a-11e8-92a9-f30c1fdfa60c.png)

**Comment by pkzxs** [comment]
@lionnexx1986  I'm really appreciated to your kindness ❤️ Thanks.

About the window position, I'm afraid it can't change. Because of the limitation of UWP, the window only have the ability to change its size, but can't set the position, I've tried to find a solution but at this time I didn't make it. If I've found out how to set the position, I'll make it ASAP.

**Comment by lionnexx1986** [comment]
@pkzxs OK i understand. Thanks for the explain! Its very...dissapointing
For more beauty UI. I recommend bigger album picture and artist picture. Then i enter to the artist and album im see the big picture, then i start scroll - picture setting smaller. Like in Music Groove Player. 
Somekind effect you have in a "Home Screen" there title "Good morning" "Good Evening" "Good morning"
For better functionality. Please make Genres in Album category and Tag Editor for Albums!

What do you think you can make it?

**Comment by pkzxs** [comment]
@lionnexx1986  
About the pictures of albums and artists, these are exactly what I will do in next several updates 😄 please give me some time to do it, it's a bit hard work.

Then about the mini player, on the UWP, the position is manged by system, there's no API to set the position, or even know where this app is placed. Groove and this app can save position is because we're going to another display mode called CompactOverlay (always on top), and the system can remember position automatically when changing to another display mode.

So, I think at this moment, there's no way to custom the window position, if I find out how to set it, I will add it immediately, thx. 😄

**Comment by lionnexx1986** [comment]
@pkzxs OK i got it, about mini player! I've found solution (Fullscreen).
I understanstand, it's very hard to work on player alone. Music Groove made by dozens of coders in big corporation. You alone made that they do.
So i very I appreciate your work!

**Comment by lionnexx1986** [comment]
Hi @pkzxs I like new artist and album pages. In albums category, then call artist submenu, title "Albums" stay on the screen. 
![2018-05-25](https://user-images.githubusercontent.com/39178626/40543555-a8756352-602d-11e8-80ad-ab5986225fd6.png)
